website/addons/s3/settings/defaults.py

```diff
@@ -4,6 +4,22 @@
 
 ALLOWED_ORIGIN = '*'
 
+BUCKET_LOCATIONS = {
```

Comment:
Better, but it still doesn't address that the default locations are in two places and must be identical. Using a JSON file should solve this.